packages/paste-website/src/components/Breadcrumb.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {themeGet} from 'styled-system';
+import {Link} from 'gatsby';
+
+const CrumbSlashStyled = styled.aside(props => ({
+  padding: `0 ${themeGet('space.space20')(props)}`,
+  display: 'inline',
+  color: themeGet('textColors.colorTextWeak')(props),
+}));
+
+const CrumbSlash: React.FC<{}> = () => <CrumbSlashStyled>/</CrumbSlashStyled>;
+
+const Breadcrumb: React.FC<{}> = ({children}) => {
+  return (
+    <>
+      {React.Children.map(children, child => (
+        <>
+          {child}
+          <CrumbSlash />
```

Comment:
Can we add a check to see not show this `<CrumbSlash />` after the last breadcrumb. Not a blocker, just a thought.

Reply:
This is what the design wanted /shrug